packages/components/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@kaizen/components",
+  "version": "1.0.0",
+  "description": "Kaizen component library",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "future",
+    "src",
+    "dist"
+  ],
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf 'dist'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git"
+  },
+  "author": "Geoffrey Chong <geoff.chong@cultureamp.com>",
+  "license": "ISC",
```

Comment:
Just curious - what's with the license change? The others say MIT

Reply:
I just didn't use the plop template and ran npm init 😅